### PR TITLE
modify Dockerfile to fix go build issue

### PR DESCRIPTION
**Pull request by sananand007**
## Description
We do not need to do a `cd main` any more as the main.go is in the working directory

## Root Cause
There was no `main` directory which was causing the `go build` step to fail in the Docker build step

## Solution
Changed the `Dockerfil` to take out the `cd main` 

## Testing
NR

## Final Checks
- [X] Have you removed commented code ?
- [ ] Have you used gofmt to format your code ?
- [ ] You have added unit tests

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,10 +1,9 @@
 FROM golang:latest as unwindtest
-RUN ls -altr
 
 RUN ls -al
```
Comment (weihesdlegend) [inline Dockerfile:4]: Is this needed?

File: Dockerfile [hunk 2]
```diff
@@ -1,10 +1,9 @@
...
 COPY . /app/
 WORKDIR /app
 
 RUN go get -v -t -d ./...
-RUN ls -altr && cd main/ && go build -v .
+RUN ls -altr &&  go build -v .
```
Comment (weihesdlegend) [inline Dockerfile:8]: Go build -v . Should be sufficient

**Review by weihesdlegend — CHANGES_REQUESTED** [review]

**Comment by sananand007** [comment]
@weihesdlegend Are all changes in ? Could we approve

**Comment by weihesdlegend** [comment]
> @weihesdlegend Are all changes in ? Could we approve

No, where is go test? Do we want to have run server part also?

**Comment by sananand007** [comment]
Go test is a separate test, look at the build logs. Do we need to have the test run the docker image? Any reasons for that

**Comment by weihesdlegend** [comment]
> Go test is a separate test, look at the build logs. Do we need to have the tests run within the docker image? Any reasons for that
> 
> ```
> function run_docker() {
> 	docker run --rm unwindtest:$BUILD_NUMBER /bin/sh -c "echo $PWD && \
> 		go test -v ./..."
> }
> ```

That's fair, but can you try to make it run under some port with `CMD`?

File: .gitignore [hunk 3]
```diff
@@ -5,3 +5,4 @@ bin/
 .idea
 .DS_Store
 lock.json
+planner-app/
```
Comment (weihesdlegend) [inline .gitignore:8]: remove this line since it does not exist on master

**Review by weihesdlegend — APPROVED** [review]
LGTM

**Pull request merged**